mlflow/pytorch/pytorch_autolog.py

```diff
@@ -0,0 +1,197 @@
+        """
+
+        def __init__(self):
+            self.early_stopping = False
+
+        def on_epoch_end(self, trainer, pl_module):
+            """
+            Log loss and other metrics values after each epoch
+
+            :param trainer: pytorch lightning model instance
+            :param pl_module: pytorch lightning base module
+            """
+            if (pl_module.current_epoch + 1) % every_n_iter == 0:
+                for key, value in trainer.callback_metrics.items():
+                    try_mlflow_log(
+                        mlflow.log_metric, key, float(value + 1), step=pl_module.current_epoch
+                    )
+
+            for callback in trainer.callbacks:
+                if isinstance(callback, pl.callbacks.early_stopping.EarlyStopping):
+                    self._early_stop_check(callback)
+
+        def on_train_start(self, trainer, pl_module):
+            """
+            Logs Optimizer related metrics when the train begins
+
+            :param trainer: pytorch lightning model instance
+            :param pl_module: pytorch lightning base module
+            """
+            mlflow.set_tag(key="Mode", value="training")
+            try_mlflow_log(mlflow.log_param, "epochs", trainer.max_epochs)
+
+            for callback in trainer.callbacks:
+                if isinstance(callback, pl.callbacks.early_stopping.EarlyStopping):
+                    self.early_stopping = True
+                    self._log_early_stop_params(callback)
+
+            if hasattr(trainer, "optimizers"):
+                for optimizer in trainer.optimizers:
+                    try_mlflow_log(mlflow.log_param, "optimizer_name", type(optimizer).__name__)
+
+                    if hasattr(optimizer, "defaults"):
+                        defaults_dict = optimizer.defaults
+                        if "lr" in defaults_dict:
+                            try_mlflow_log(mlflow.log_param, "learning_rate", defaults_dict["lr"])
+
+                        if "eps" in defaults_dict:
+                            try_mlflow_log(mlflow.log_param, "epsilon", defaults_dict["eps"])
+
+            summary = str(ModelSummary(pl_module, mode="full"))
+            tempdir = tempfile.mkdtemp()
+            try:
+                summary_file = os.path.join(tempdir, "model_summary.txt")
+                with open(summary_file, "w") as f:
+                    f.write(summary)
+
+                try_mlflow_log(mlflow.log_artifact, local_path=summary_file)
+            finally:
+                shutil.rmtree(tempdir)
+
+        def on_train_end(self, trainer, pl_module):
+            """
+            Logs the model checkpoint into mlflow - models folder on the training end
+
+            :param trainer: pytorch lightning model instance
+            :param pl_module: pytorch lightning base module
+            """
+
+            mlflow.pytorch.log_model(pytorch_model=trainer.model, artifact_path="models")
```

Comment:
```suggestion
            mlflow.pytorch.log_model(pytorch_model=trainer.model, artifact_path="model")
```

We usually use "model" when logging a model in auto-logging.

Reply:
i changed it to `model`